taskt/Core/Automation/Commands/Variable Commands/SetVariableIndexCommand.cs

```diff
@@ -1,111 +1,87 @@
             }
 
             if (requiredVariable != null)
             {
-
                 var index = int.Parse(v_Index.ConvertToUserVariable(sender));
-
                 requiredVariable.CurrentPosition = index;
-          
             }
             else
             {
-                throw new Exception("Attempted to update variable index, but variable was not found. Enclose variables within brackets, ex. {vVariable}");
+                throw new Exception("Attempted to update variable index, but variable was not found. Enclose variables within braces, ex. {vVariable}");
             }
         }
 
-        private Script.ScriptVariable LookupVariable(Core.Automation.Engine.AutomationEngineInstance sendingInstance)
-        {
-            //search for the variable
-            var requiredVariable = sendingInstance.VariableList.Where(var => var.VariableName == v_userVariableName).FirstOrDefault();
-
-            //if variable was not found but it starts with variable naming pattern
-            if ((requiredVariable == null) && (v_userVariableName.StartsWith(sendingInstance.EngineSettings.VariableStartMarker)) && (v_userVariableName.EndsWith(sendingInstance.EngineSettings.VariableEndMarker)))
-            {
-                //reformat and attempt
-                var reformattedVariable = v_userVariableName.Replace(sendingInstance.EngineSettings.VariableStartMarker, "").Replace(sendingInstance.EngineSettings.VariableEndMarker, "");
-                requiredVariable = sendingInstance.VariableList.Where(var => var.VariableName == reformattedVariable).FirstOrDefault();
-            }
-
-            return requiredVariable;
-        }
-
-        public override string GetDisplayValue()
-        {
-            return base.GetDisplayValue() + " [Update Variable '" + v_userVariableName + "' index to '" + v_Index + "']";
-        }
-
-        public override List<Control> Render(UI.Forms.frmCommandEditor editor)
+        public override List<Control> Render(frmCommandEditor editor)
         {
             //custom rendering
             base.Render(editor);
 
-
             //create control for variable name
-            RenderedControls.Add(CommandControls.CreateDefaultLabelFor("v_userVariableName", this));
-            var VariableNameControl = CommandControls.CreateStandardComboboxFor("v_userVariableName", this).AddVariableNames(editor);
-            RenderedControls.AddRange(CommandControls.CreateUIHelpersFor("v_userVariableName", this, new Control[] { VariableNameControl }, editor));
+            RenderedControls.Add(CommandControls.CreateDefaultLabelFor("v_VariableName", this));
+            var VariableNameControl = CommandControls.CreateStandardComboboxFor("v_VariableName", this).AddVariableNames(editor);
+            RenderedControls.AddRange(CommandControls.CreateUIHelpersFor("v_VariableName", this, new Control[] { VariableNameControl }, editor));
```

Comment:
Please replace this custom rendering with a "CreateDefaultInputGroupFor"

Reply:
done